Source/WebCore/Modules/webxr/WebXRSystem.cpp

```diff
@@ -558,10 +559,10 @@ void WebXRSystem::unregisterSimulatedXRDeviceForTesting(PlatformXR::Device& devi
 
     ASSERT(m_testingDevices);
     bool removed = m_immersiveDevices.remove(device);
-    ASSERT_UNUSED(removed, removed || m_inlineXRDevice == &device);
-    if (m_activeImmersiveDevice == &device)
+    ASSERT_UNUSED(removed, removed || m_inlineXRDevice.get().get() == &device);
+    if (m_activeImmersiveDevice.get().get() == &device)
```

Comment:
Do we really need the second .get()? It is a bit ugly.

Reply:
Ah, `RefPtr<>::operator==()` is overloaded to compare to a raw pointer, so we can remove the second `.get()`.

Reply:
Will fix in the third patch (about to upload).